cnv/cnv-attaching-vm-multiple-networks.adoc

```diff
@@ -0,0 +1,28 @@
+[id="attaching-to-multiple-networks"]
+= Attaching a virtual machine to multiple networks
+include::modules/cnv_document_attributes.adoc[]
+:context: cnv-attaching-multiple-networks
+toc::[]
+
+{ProductName} provides Layer-2 networking capabilities that allow you to connect
+virtual machines to multiple networks. You can import virtual machines with
+existing workloads that depend on access to multiple interfaces. You can also
+configure a PXE network so that you can boot machines over the network.
+
+To get started, a network administrator configures a NetworkAttachmentDefinition
+of type `cnv-bridge`. Then, users can attach Pods, VMIs, and VMs to the
+bridge network. From the {ProductName} web console, you can create a vNIC
+that refers to the bridge network.
+
+include::modules/cnv-networking-glossary.adoc[leveloffset=+1]
+
+include::modules/cnv-connecting-resource-bridge-network.adoc[leveloffset=+1]
+
+[CAUTION]
```

Comment:
Isn't "Caution" a bit harsh for this comment?

Reply:
Yeah, I think it would be fine as a [NOTE]. I'll change it.

Reply:
Changed here: https://github.com/openshift/openshift-docs/pull/14351/commits/76f810354b4a29b3177ff0173d55e3894bdfaa6a